.codecatalyst/actions/action.yml

```diff
@@ -3,43 +3,43 @@ Name: 'SonarCloudScan'
 Version: '0.0.0'
 Description: 'Scan your code with SonarCloud to detect bugs, vulnerabilities, and code smells in 26+ programming languages.'
 Configuration:
-  SonarProjectBaseDir:
-    Description: 'Sonar Project Base Directory'
-    Required: true
-    Default: '.'
-    DisplayName: 'Sonar Project Base Directory'
-    Type: string
-  SonarProjectKey:
-    Description: 'Project Key'
-    Required: false
-    DisplayName: 'Project Key'
-    Type: string
-  SonarOrganization:
-    Description: 'Organization'
-    Required: false
-    DisplayName: 'Organization'
-    Type: string
-  SonarAdditionalArgs:
-    Description: 'Additional arguments'
-    Required: false
-    DisplayName: 'Additional arguments'
-    Type: string
-  SonarToken:
-    Description: 'Sonar Token'
-    Required: true
-    DisplayName: 'Sonar Token'
-    Type: string
-  SonarBranchName:
-    Description: 'Branch Name'
-    Required: false
-    Default: '${WorkflowSource.BranchName}'
-    DisplayName: 'Branch Name'
-    Type: string
+    SonarProjectBaseDir:
```

Comment:
Any idea why all these files not related to Cirrus build have changed? This will be an issue to merge to the main branch later.

Reply:
Actually, it's related to this commit from codecatalyst from the base branch https://github.com/SonarSource/sonarcloud-codecatalyst-action/commits/snapshot-for-ivan
https://github.com/SonarSource/sonarcloud-codecatalyst-action/commit/315a2fa7ee7fd54106bf42d94f057cde9663882c

Reply:
Ah ok. Then it is due to copy/paste between the 2 repos... Not good. We must be careful for the next versions.
Do you want to remove them from the PR ?